### ISSUE TITLE
Selection Criteria for Training Epochs

### ISSUE THREAD

**Issue by ldkong1205** [fanq15/GCoNet#8]
The training loss for GCoNet trained by DUTS_class still drops near 50 training epochs. I have the following questions:

(1) Are there any selection criteria that you choose 50 epochs for training?
(2) Will the performance be better if training for more than 50 epochs? (I notice that GCoNet shares a lot of similarities with GICD, which originally trained for 100 epochs)
(3) Are you using any validation dataset to select hyperparameters? (E.g., GICD uses CoSal2015 for validation during training)

Thanks for the work and looking forward to your reply.

**Comment by fanq15** [comment]
Hi, thanks for your interest. for Your questions:
(1) There are no selection criteria for the training epoch. We just found that 50 epochs is enough for a good performance.
(2) 50 epochs is enough and you can try 100 epochs.
(3) We use CoCA, CoSOD3k and Cosal2015 in our validation experiments and we mainly focus on the CoCA dataset.

**Comment by ldkong1205** [comment]
Got it. Many thanks.
